app/helpers/stream_entries_helper.rb

```diff
@@ -51,7 +51,9 @@ def rtl?(text)
     rtl_characters = /[\p{Hebrew}|\p{Arabic}|\p{Syriac}|\p{Thaana}|\p{Nko}]+/m.match(text)
 
     if rtl_characters.present?
-      total_size = text.strip.size.to_f
+      # Remove mentions before counting characters to decide RTL ratio
```

Comment:
ok, I'll drop it from the PR. we'll use it on our instance till I find the time to study this.